shared/js/shared-utils/parse-user-agent-string.es6.js

```diff
@@ -0,0 +1,24 @@
+module.exports = (uaString) => {
+    if (!uaString) uaString = window.navigator.userAgent
+
+    let browser
+    let version
+
+    try {
+        const rgx = uaString.match(/(Firefox|Chrome|Safari)\/([0-9]+)/)
```

Comment:
@andrey-p what do you think about renaming rgx to browserVersionRgx or something like that?